packages/flutter/lib/src/material/progress_indicator_theme.dart

```diff
@@ -66,6 +70,18 @@ class ProgressIndicatorThemeData with Diagnosticable {
   /// {@macro flutter.material.RefreshProgressIndicator.backgroundColor}
   final Color? refreshBackgroundColor;
 
+  /// Overrides the border radius of the [ProgressIndicator].
+  final BorderRadiusGeometry? borderRadius;
+
+  /// Overrides the stop indicator color of the [ProgressIndicator].
```

Comment:
Seems these new properties are specifically for `LinearProgressIndicator`. If so, maybe we can make it clear here:)